Specs/Scene/GltfStructuralMetadataLoaderSpec.js

```diff
@@ -519,32 +519,34 @@ describe(
         resource: schemaResource,
       });
 
-      if (rejectPromise) {
-        schemaCopy.promise.catch(function () {});
-      }
-
-      featureMetadataLoaderCopy.load();
+      structuralMetadataLoaderCopy.load();
 
-      return waitForLoaderProcess(featureMetadataLoaderCopy, scene).then(
-        function () {
-          // Ignore featureMetadataLoaderCopy destroying its buffer views
+      return waitForLoaderProcess(structuralMetadataLoaderCopy, scene).then(
```

Comment:
I'm not sure if I did this properly, I see that in `main` they were trying to remove when...